Formula/o/openai-whisper.rb

```diff
@@ -27,7 +27,7 @@ class OpenaiWhisper < Formula
   depends_on "pytorch"
 
   on_linux do
-    depends_on "pkg-config" => :build
+    depends_on "pkgconf" => :build
```

Comment:
Could probably make this unconditional given `rust` + shims make `pkg-config` available during the build on macOS already anyway.

Reply:
Sure.

On side note, I am probably going to rewrite the `pkg-config` shim to require explicit dependency on macOS rather than hardcoded path